.pylintrc

```diff
@@ -426,7 +426,7 @@ max-public-methods=40
 max-returns=6
 
 # Maximum number of statements in function / method body
-max-statements=50
+max-statements=75
```

Comment:
Unless we really agree that the rules in this file should change, I'd rather we first try to adjust the code to conform (in this case, break up the offending function into smaller functions), then if that's not possible override the rule just for that function. That way we're being intentional about exceptions while still encouraging small modular code everywhere else.